csv_dump.py

```diff
@@ -74,27 +74,60 @@ def query_and_write_data(cur, table_name, base_path):
         base_path (str): Base path of where the CSV file dump of the table is to be stored.
                         (Example: /example/path/to/desired/directory/)
     '''
-    query = "COPY (SELECT * FROM {}) TO STDOUT  DELIMITER ',' CSV HEADER;".format(table_name)
-    file_path = "{}{}/{}.csv".format(base_path, table_name, date.today().strftime("%Y_%m_%d"))
+    table_name, table_query, need_params, create_temp_table = table_query_info
```

Comment:
Given the number of arguments, I'm going to ask for a separate utility class so that we avoid brittleness.

Reply:
That's a good point. The queries and arguments may change depending on arguments that may come in future. I would rather spend time refactoring when we have a clear understanding down the road with the remodeled schema. It's something I'm planning to tackle in the coming sprints.

Reply:
It'll be easier to refactor against a class rather than a n-ple. We can and will need to refactor later. I also don't think it's worth really refactoring too heavily yet.